src/test/functional/test.js

```diff
@@ -0,0 +1,37 @@
+import { ReactSelector, waitForReact } from 'testcafe-react-selectors';
+import UserInfo from '../../stores/user-info';
+
+fixture`App tests`
+    .page(`http://localhost:3000`)
+    .beforeEach(async () => {
+        await waitForReact();
+    });
+
+test('Update user', async t => {
```

Comment:
We can divide this test in two parts:
1) successful editing (`Update user`)
2)unsuccessful editing (`User can be updated with an invalid phone number`)